sbin/build.sh

```diff
@@ -1802,15 +1803,20 @@ getFirstTagFromOpenJDKGitRepo() {
     TAG_SEARCH="aarch64-shenandoah-jdk8u*-b*"
   fi
 
-  # If openj9 and the closed/openjdk-tag.gmk file exists which specifies what level the openj9 jdk code is based upon,
-  # read OPENJDK_TAG value from that file.
-  local openj9_openjdk_tag_file="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/closed/openjdk-tag.gmk"
-  if [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_OPENJ9}" ]] && [[ -f "${openj9_openjdk_tag_file}" ]]; then
-    firstMatchingNameFromRepo=$(grep OPENJDK_TAG ${openj9_openjdk_tag_file} | awk 'BEGIN {FS = "[ :=]+"} {print $2}')
+  if [ "${BUILD_CONFIG[OPENJDK_FOREST_DIR]}" == "true" ]; then
```

Comment:
> Don't modify the getFirstTagFromOpenJDKGitRepo() method at all, it should only be called for a "git repo", instead you need to modify the getOpenJdkVersion() to correctly return the jdk tag version of the forest openjdk

Thanx, will do!

> source, I suspect as specified via the build arg --tag <jdk tag version>. So I am thinking if --openjdk-source <path to tarball> is specified then --tag <jdk tag version> is mandatory in order to specify what the version within the source tarball is being built... The tag version string is used by the build scripts for things like the image archive filenames...

I think it defaults to some unknonw or head, but I will double check. If it do not default, then I would rather default it to some "local" rather then make it mandatory. Of course, it it is set, then it will be used